apps/passport-server/resources/email/one-click-email/email.txt

```diff
@@ -0,0 +1,15 @@
+The Edge Esmeralda team and community is excited to welcome you to Healdsburg!
+
+Click here to access your ticket & QR code:
+
+{{oneClickLoginLink}}
+
+When you arrive in Healdsburg, use your QR code to check in and receive your wristband. Check-in is at the Hotel Trio lobby:
+
+https://www.google.com/maps/place/Hotel+Trio+Healdsburg/@38.6255211,-122.8748975,15z/data=!4m9!3m8!1s0x808410c772dd18c5:0xfcde629c363bbd48!5m2!4m1!1i2!8m2!3d38.6255211!4d-122.8748975!16s%2Fg%2F11f54sz8gb?sa=X&ved=1t:2428&ictx=111&entry=tts&g_ep=EgoyMDI0MDYwNC4wKgBIAVAD
+
+The check-in schedule can be viewed here:
+
+https://blog.edgeesmeralda.com/p/check-in-at-edge-esmeralda
+
+Can't wait to see you soon!
```

Comment:
nit: is there a reason why the email text is purple in the screenshot on your PR?

Reply:
gmail seems to format emails i deems to be 'duplicates' in this purple way, and the screenshot i posted was like the 3rd email in a chain of these 'duplicates'